packages/cli/src/runners/task-runner-process.ts

```diff
@@ -61,10 +65,18 @@ export class TaskRunnerProcess extends TypedEmitter<TaskRunnerProcessEventMap> {
 		'NODE_FUNCTION_ALLOW_EXTERNAL',
 	] as const;
 
+	private get isInternalMode() {
```

Comment:
This check shouldn't be needed, as this class is only ever loaded if we are in internal mode. See `TaskRunnerModule`